src/objects/npe03__Recurring_Donation__c.object

```diff
@@ -22,6 +22,42 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>CardExpirationMonth__c</fullName>
+        <description>The expiration month of the card used for this Recurring Donation.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The expiration month of the card used for this Recurring Donation.</inlineHelpText>
+        <label>Card Expiration Month</label>
+        <length>2</length>
+        <required>false</required>
+        <trackTrending>false</trackTrending>
+        <type>Text</type>
```

Comment:
Minor thing, but if we make these fields a number type, they can be sorted easier in reports.

@RadSFDeveloper @force2b

Reply:
I followed the field definition as it is on the Payment SObject , for example [`Card_Expiration_Month__c`](https://github.com/SalesforceFoundation/NPSP/blob/master/src/objects/npe01__OppPayment__c.object#L104). To be consistent, let's follow what has already been done.

Reply:
Got it - sounds good.